src/StudioCore/Assets/Paramdex/ER/Defs/GameSystemCommonParam.xml

```diff
@@ -2193,45 +2197,67 @@
       <Increment>0.1</Increment>
       <SortID>40001</SortID>
     </Field>
+    
     <Field Def="dummy8 reserved_124[48]" RemovedVersion="10701000">
       <DisplayName>予約</DisplayName>
       <Description>(dummy8)</Description>
       <SortID>70003</SortID>
     </Field>
-    <Field Def="dummy8 reserved_124[32]" FirstVersion="10701000">
+    
+    <Field Def="dummy8 reserved_124[32]" FirstVersion="10701000" RemovedVersion="11210015">
```

Comment:
Honestly it's good to get this mapped out.
I think with more effort we'll find we can bump back these numbers further - if they were added in 1.07 they probably were in use and should've been listed as unks instead of dummy8. Such is the hasty approach cus I'm pretty sure I'm the idiot who did it.